db/redis.js

```diff
@@ -79,6 +87,29 @@ function addXeThruSensorData(locationid, state, rpm, distance, mov_f, mov_s) {
 }
 /* eslint-enable function-call-argument-newline */
 
+async function addInnosentRadarSensorData(locationid, inPhase, quadrature) {
+  const inPhaseArray = inPhase.split(',')
+  const quadratureArray = quadrature.split(',')
+  const pipeline = client.pipeline()
+  for (let i = 0; i < inPhaseArray.length; i += 1) {
+    pipeline.xadd(`innosent:${locationid}`, '*', 'inPhase', inPhaseArray[i], 'quadrature', quadratureArray[i])
+  }
+  await pipeline.exec(err => {
+    helpers.log(err)
+  })
+}
+
+async function getInnosentWindow(locationID, startTime, endTime, windowLength) {
+  const rows = await client.xrevrange(`innosent:${locationID}`, startTime, endTime, 'count', windowLength)
```

Comment:
Are these `await`s in this file also susceptible to the unhandle promise rejection warning? Maybe they need a try-catch.